include/boost/redis/connection.hpp

```diff
@@ -174,7 +174,6 @@ struct connection_impl {
       stream_.close();
       writer_timer_.cancel();
       receive_channel_.cancel();
-      mpx_.cancel_on_conn_lost();
    }
 
    bool is_open() const noexcept { return stream_.is_open(); }
```

Comment:
Can we deprecate `is_open()` as well? Or at least is should be renamed to `is_healthy()`.